tests/wpt/web-platform-tests/html/semantics/forms/textfieldselection/selection-not-application.html

```diff
@@ -7,45 +7,90 @@
 <script src="/resources/testharnessreport.js"></script>
 <div id="log"></div>
 <script>
-  var types = ["hidden", "email", "datetime-local", "date", "month", "week", "time", "number", "range", "color", "checkbox", "radio", "file", "submit", "image", "reset", "button"]; //types for which the API doesn't apply
-  var types2 = ["text", "search", "tel", "url", "password"]; //types for which the API applies
+  var nonApplicableTypes = ["hidden", "email", "datetime-local", "date", "month", "week", "time", "number", "range", "color", "checkbox", "radio", "file", "submit", "image", "reset", "button"];
+  var applicableTypes = ["text", "search", "tel", "url", "password", "aninvalidtype", null];
```

Comment:
Why are `null` and `"aninvalidtype"` accepted? Is it because they're both going to be treated as `"text"`?

Reply:
Correct. My first implementation only worked with the former, but now since I've done the `InputType` refactoring, both are correctly treated as `"text"`.